package/boot/uboot-envtools/files/layerscape

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+#
+# Copyright (C) 2016 LEDE
+#
+
+[ -e /etc/config/ubootenv ] && exit 0
+
+touch /etc/config/ubootenv
+
+. /lib/uboot-envtools.sh
+. /lib/functions.sh
+
+board=$(board_name)
+
+case "$board" in
+	traverse,ls1043v | \
+	traverse,ls1043s)
+		ubootenv_add_uci_config "/dev/mtd1" "0x40000" "0x2000"  "0x20000"
+	;;
```

Comment:
Not sure I understand the lack of a default case...  do you want to proceed even when you don't match either of those two?

Reply:
Good question, the structure here is exactly the same as the other archs in uboot-envtools/files/

I think the intention was that a config for an unknown board in UCI will be retained (by "config_foreach ubootenv_add_app_config ubootenv" underneath), that said, I'm not sure why you would inject it elsewhere. 

Probably warrants a investigation of the uboot-envtools package itself.